svelte-kit-scss/src/routes/test/+page.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
```

Comment:
Ha!  Yes, this is where storybook would be a very good use, so you don't have to make pages like this